gensim/utils.py

```diff
@@ -490,6 +497,13 @@ def save(self, fname_or_handle, separately=None, sep_limit=10 * 1024**2,
         in both Python 2 and 3.
 
         """
+        import os
```

Comment:
Imports belong to the top of file (separated into builtin - external - internal blocks).

(here and elsewhere)